src/data/data.ts

```diff
@@ -125,69 +138,138 @@ const livingAloneMult =
   Interaction.oneTime.multiplier *
   Setting.indoor.multiplier
 
-export const RiskProfile: { [key: string]: FormValue } = {
+const SYMPTOM_FREE_LAST_SEEN_TODAY_MULT = 0.5
+const SYMPTOM_FREE_LAST_SEEN_MORE_THAN_THREE_DAYS_AGO = 1 / 7
+
+export const personRiskMultiplier: (arg: {
```

Comment:
So is this whole thing basically a refactor with some additional code architecture for future features? I wanna understand before I give it a thumbs up.

Reply:
Correct. Before, person risks were opaque blobs of multiplication and addition. When I added `housemateMultiplier` it doubled down on this mess - two blobs of multiplication and addition that weren't obviously tied together. 

Right now, this pays down that debt - RiskProfile contains information about the # of housemates and traceable contacts for a risk profile, and all the multiplications and additions are handled by personRiskMultiplier. Therefore, if you trust  the logic of `personRiskMultiplier` (which, conveniently, is testable), then it becomes much easier to read and verify new RiskProfile categories, and it programmatically handles the case where the user is a housemate of the person in question.

The backend for the proposed additional feature this allows is included in personRiskMultiplier - you can specify that the contact and their contacts are verified symptom free to reduce their risk, which is a big player in making interactions safer (almost 4x for many categories).

Theoretically you could also use this to give credit for contact tracing (but the mechanics of asking for this are weird).

Happy to talk you through the change more if you'd like.